aten/src/ATen/native/cuda/Normalization.cuh

```diff
@@ -694,6 +694,8 @@ void batch_norm_stats_cuda_template(
   Tensor dummy_var_;
   auto input_reshaped = input_.reshape({input_.size(0), input_.size(1), -1}); // internally we merge the feature dimensions
 
+  resize_output(out_mean, {n_input});
```

Comment:
why do we need this?

Reply:
Not strictly necessary for this change, but I noticed the out-argument variants didn't resize outputs correctly.